src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderMovementResourceApiTest_moveToCell.java

```diff
@@ -135,14 +127,6 @@ public void livingUnitId_inactive_badRequest() {
         assertThat(getBodyAsJsonContent(response)).extractingJsonPathStringValue("$.userMessage").isEqualTo(livingUnitNotFoundError);
     }
 
-    @Test
-    public void livingUnitId_missing_notFound() {
-        final var response = testRestTemplate.exchange("/api/bookings/1/living-unit/?reasonCode=ADM&dateTime=2020-03-24T13:24:35", PUT, anEntity(), String.class);
```

Comment:
this was then defaulting to the global error handler so not actually calling this endpoint.  Pointless test.